pkg/services/rendering/rendering.go

```diff
@@ -110,6 +111,19 @@ func (rs *RenderingService) RenderErrorImage(err error) (*RenderResult, error) {
 }
 
 func (rs *RenderingService) Render(ctx context.Context, opts Opts) (*RenderResult, error) {
+	startTime := time.Now()
+	result, err := rs.render(ctx, opts)
+	if err != nil {
+		metrics.MRenderingRequestFailed.Inc()
+	} else {
+		metrics.MRenderingRequestCompleted.Inc()
+	}
+	elapsedTime := time.Since(startTime).Nanoseconds() / int64(time.Millisecond)
```

Comment:
Isn't better just use `Milliseconds()`?
```suggestion
	elapsedTime := time.Since(startTime).Milliseconds()
```

Reply:
Think Milliseconds was added in recent Go versions.